src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesList = sourceString
```

Comment:
Consider adding validation to ensure `sourceString` is not null or undefined before calling `.split(';')`. This will prevent runtime errors if the input is not as expected.